data/gameserverlauncher.ini

```diff
@@ -7,6 +7,9 @@ dir = C:\Games\Tribes_Ascend_Parting_Gifts\Binaries\Win32
 controller_dll = TAMods-Server.dll
 controller_config = data\gamesettings\ootb\serverconfig.lua
 
+# To set a custom game server log directory
+# log_dir = C:\CustomLogDir
```

Comment:
Let's put it above controller_dll and under dir so that the controller_config related content is together.